corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/collections.js

```diff
@@ -9,7 +9,7 @@ FormplayerFrontend.module("Users.Collections", function(Collections, FormplayerF
             if (!this.domain) {
                 throw new Error('Cannot instantiate collection without domain');
             }
-            return '/a/' + this.domain + '/settings/users/commcare/';
+            return '/a/' + this.domain + '/cloudcare/api/login_as/users/';
```

Comment:
this used to hit the mobile workers route. this now hits a separate view which frees us up to add functionality without affecting the users page. the big one being changing permissions